tests/msgpack.light.tests/Generators/InterfaceInheritanceTest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Shouldly;
+
+using Xunit;
+
+namespace ProGaudi.MsgPack.Light.Tests.Generators
+{
+    public class InterfaceInheritanceTest : MapGeneratorTestBase, IClassFixture<InterfaceInheritanceFixture>
```

Comment:
Should we add tests to check that generated converter for interface with inheritance works the same as created manually?